imops/measure.py

```diff
@@ -120,7 +120,7 @@ def label(
         res.append(('labels', np.array(range(1, num_components + 1))))
```

Comment:
is this true in case when there is no background?

Reply:
Yeah, labels are started with 1. There are also check in `test_measure.test_no_background`, `test_measure.test_no_background2`